### PR TITLE
Update install_yamls_setup playbook

### PR DESCRIPTION
**Pull request by weinimo**
Attempt to update the playbook for setting up a development environment using install_yaml.

### CODE REVIEW

File: api/bases/octavia.openstack.org_octaviaamphoracontrollers.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```
Comment (stuggi) [inline api/bases/octavia.openstack.org_octaviaamphoracontrollers.yaml:6]: it seems you are using an old controller-gen version. remove the `bin/controller-gen` and rerun the make generate manifests

Comment (weinimo) [reply api/bases/octavia.openstack.org_octaviaamphoracontrollers.yaml]: Thanks!

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**